tests/flow/test_config.py

```
@@ -307,6 +307,12 @@ def test09_set_invalid_values(self):
             except redis.exceptions.ResponseError as e:
                 assert(("Failed to set config value %s to invalid" % config) in str(e))
 
+        try:
+            self.redis_con.execute_command("GRAPH.CONFIG", "SET", "ASYNC_DELETE", 1)
+            assert(False)
```

Ruff (0.15.14)

[warning] Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)